WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -445,6 +470,23 @@ public void onEventMainThread(NotificationEvents.NotificationsChanged event) {
         mTabLayout.checkNoteBadge();
     }
 
+    @SuppressWarnings("unused")
+    public void onEventMainThread(ConnectionChangeReceiver.ConnectionChangeEvent event) {
+        updateConnectionBar(event.isConnected());
+    }
+
+    private void checkConnection() {
+        updateConnectionBar(NetworkUtils.isNetworkAvailable(this));
+    }
+
+    private void updateConnectionBar(boolean isConnected) {
+        if (isConnected && mConnectionBar.getVisibility() == View.VISIBLE) {
+            AniUtils.animateBottomBar(mConnectionBar, false);
```

Comment:
When we're animating the connection bar can we apply the animation to the FAB as well? When testing the tap-to-retry functionality I noticed that it jumps as the connection bar slides in/out.